### ISSUE TITLE
1.6.2 is crashing the game when using optifine 1.16.3 HD U G3

### ISSUE THREAD

**Issue by LDH23M** [Chocohead/OptiFabric#61]
When i put only the optifabric in the mods folder the game starts and gives me a error that optifine is not installed that is correct but as soon as i plase optifine 1.16.3 HD U G3 in the mods folder the game doesnt open it just comes back with a game crash error 0.

**Comment by geogdog** [comment]
I have the same issue.  Using the following mods:
```
OptiFine_1.16.3_HD_U_G3.jar
fabric-api-0.21.0+build.407-1.16.jar
fabric-carpet-1.16.3-1.4.11+v200910.jar
litematica-fabric-1.16.3-0.0.0-dev.20200913.215807.jar
malilib-fabric-1.16.3-0.10.0-dev.21+arne.1.jar
minihud-fabric-1.16.3-0.19.0-dev.20200912.222204.jar
optifabric-1.6.2.jar
```
Although, when I remove all the other mods apart from fabic-api, optifabric and Optifine, it loads without crashing.

For me it seems like the carpet mod is what's making it crash.

**Comment by CyberFeast2K** [comment]
Same issue.

Edit But I have the latest version of optifabric 1.6.4. Here’s the crash log. [https://pastebin.com/CztkF2Tx](url)

**Comment by Chocohead** [comment]
This was fixed in 1.6.4, see #59
